tests/escrow/package-lock.json

```diff
@@ -0,0 +1,5396 @@
+{
```

Comment:
Please remove the package-lock. The repo is not committing them for the examples.

Reply:
Fixed